gui/packages/ubuntupro/lib/core/agent_api_paths.dart

```diff
@@ -19,8 +19,8 @@ String? agentAddrFilePath(String filename) {
 
 enum AgentAddrFileError { nonexistent, isEmpty, formatError, accessDenied }
 
-/// Reads the agent port from the addr file located at the full path [filepath].
-Future<Either<AgentAddrFileError, int>> readAgentPortFromFile(
+/// Reads the agent host and port from the addr file located at the full path [filepath].
+Future<Either<AgentAddrFileError, (String, int)>> readAgentPortFromFile(
```

Comment:
I'm not a big fan of tuples but this seems a very good use case for them, as having a class for that would be an overkill since there is no invariants between the port number and the host String. 👍🏽